docs/getting-started/config/database.md

```diff
@@ -68,17 +68,27 @@ That's enough database administration to get started. Type `\q`, followed by
 pressing the enter key. Then again type `exit` and press enter. Next, you need
 to install and configure the client.
 
-### 2. Configuring Backstage `pg` Client
+### Docker Alternative
```

Comment:
This is awesome and I think really useful, I was wondering how we feel about also suggesting a `docker compose` file? We use it and its so easy 😄  e.g.

```yaml
version: '4'
services:
  postgres:
    image: postgres:15.5-alpine
    environment:
      POSTGRES_USER: localpostgres
      POSTGRES_PASSWORD: **********
    ports:
      - 5432:5432
 ```

Then you just `up` or `down` when ever you need it.

But this is also fine, I think either way is super helpful 👏

Reply:
Cool, I've adjusted the structure to accommodate this a bit more easily IMO. Can you give this another look please?